configure.ac

```diff
@@ -70,6 +70,7 @@ AC_ARG_ENABLE([direct],
 
 dnl Checks for programs
 AC_PROG_CC_C99
+AM_PROG_CC_C_O
```

Comment:
It's worth caring about, but I think the fix is just to remove our usage of `AM_PROG_CC_C_O` from `configure.in`.  Ben, can you try making that change and building a fresh copy of libfabric on one of our RHEL6 nodes that has automake-1.11.1?  Note that you'll probably have to `module unload cisco/autotools` and check with `automake --version` to make sure you are really testing with the stock autotools from the RHEL distro.  Assuming we don't get any warnings from automake about omitting that macro, then that should be the easy fix.

If automake complains about missing `AM_PROG_CC_C_O` for versions <1.14 but complains about its presence for versions >=1.14 then we'll have to do something trickier or require 1.14 as the minimum (undesirable, IMO, but not the end of the world).